nixos/modules/programs/gamescope.nix

```diff
@@ -0,0 +1,64 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  cfg = config.programs.gamescope;
+
+  gamescope = pkgs.gamescope.override {
```

Comment:
Won't this require us to recompile the package? That doesn't seem like a great user experience. 

I think the best solution is double wrapping. That's also not great, but its better than having to recompile the program whenever you change one of the flags.

Reply:
I considered this, originally I decided it was alright because the package is actually surprisingly light to build, but I can see your point. Perhaps we should double wrap :thinking:

Reply:
I can understand that, though for some shell arguments I'd argue any time spend compiling is too much.